src/index.js.flow

```diff
@@ -0,0 +1,403 @@
+// @flow
+
+declare type SeedValue<S, V> = { seed: S, value: V };
+declare type TimeValue<V> = { time: number, value: V };
+
+declare type CreateGenerator<A> = (...args: Array<any>) => Generator<A|Promise<A>, any, any>;
+
+export interface Sink<A> {
```

Comment:
One last thing: I'd like to go with types unless there's a compelling reason to prefer interfaces.

We've [preferred `type` over `interface`](https://github.com/mostjs/core/blob/master/packages/types/index.js.flow) in `@most/core`'s flow types.  I don't see any benefit to having 2 things, types and interfaces, vs having only types.  It'd also nice to be consistent with `@most/core`, although I'm not too concerned about it.

Otherwise, this all looks good to me.